content/c-sharp/concepts/deque/deque.md

```diff
@@ -0,0 +1,109 @@
+---
+Title: 'Deque'
+Description: 'A deque is a type of data structure that allows insert and delete elements at both ends.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Deques'
+  - 'Data Structures'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
```

Comment:
```suggestion
  - 'learn-c-sharp'
  - 'paths/computer-science'
```
Include only those paths that are already present in the [catalog-content.md](documentation/catalog-content.md) and if you want the relevant course/paths to be included then you can add one. Here `learn-c-sharp` is not already available in the catalog-content.md file you can add it.